tools/clang-format/package.json

```diff
@@ -1,9 +1,5 @@
 {
-  "name": "node-core-clang-format",
-  "version": "1.0.0",
-  "description": "Formatting C++ files for Node.js core",
-  "license": "MIT",
```

Comment:
Why this was removed?

Reply:
Because this isn't a package, so it doesn't make sense to have a package name, it may confuse users.

Reply:
I just think the name, version, and license are misleading, as there isn't any source code here.

Reply:
@marco-ippolito I've re-added the description, do you want me to add the rest?